contracts/rbn-staking/FeeCustody.sol

```diff
@@ -218,11 +220,10 @@ contract FeeCustody is Ownable {
 
       // Approximate claimable by multiplying
       // current asset balance with current asset price in USD
-      claimable += balance
-        .mul(uint256(oracle.latestAnswer()))
-        .mul(_allocPCT)
-        .div(10**8)
-        .div(TOTAL_PCT);
+      claimable +=
+        (balance * uint256(oracle.latestAnswer()) * _allocPCT) /
```

Comment:
How would we handle wstETH here? There are no chainlink feeds for the asset

Reply:
we just approximate with eth feed in its stead. 1.001607 vs eth